crates/env/Cargo.toml

```diff
@@ -50,6 +50,26 @@ secp256k1 = { version = "0.24", features = ["recovery", "global-context"], optio
 rand = { version = "0.8", default-features = false, features = ["alloc"], optional = true }
 scale-info = { version = "2", default-features = false, features = ["derive"], optional = true }
 
+contract-metadata = "2.0.0-alpha.2"
+impl-serde = { version = "0.3.1", default-features = false }
+jsonrpsee = { version = "0.14.0", features = ["ws-client"] }
+pallet-contracts-primitives = "6.0.0"
+serde = { version = "1.0.137", default-features = false, features = ["derive"] }
+serde_json = "1.0.81"
+tokio = { version = "1.18.2", features = ["rt-multi-thread"] }
+log = "0.4"
+env_logger = "0.8"
+subxt = "0.24.0"
+
+# Substrate
+sp-rpc = "6.0.0"
+sp-core = "6.0.0"
+sp-keyring = "6.0.0"
+sp-runtime = "6.0.0"
+
+# TODO(#xxx) `smart-bench_macro` needs to be forked.
+smart-bench-macro = { git = "https://github.com/paritytech/smart-bench", branch = "cmichi-ink-e2e-test-mvp", package = "smart-bench-macro"}
+
```

Comment:
I think all of these new e2e dependencies should be `optional = true`, since the `e2e` code is conditional for `std`.

Arguably we may even want an additional feature to prevent them being compiled for the metadata generation which is `std`, there are some hefty dependencies here.